2023/Nwajei Edgar/README.md

```diff
@@ -0,0 +1,19 @@
+I’m Edgar, a frontend dev, and also a Technical writer. I started my journey not quite long ago, and I heard about Opensource during personal research on how to expand my knowledge and connect with others. That's when I found Outreachy and applied. Now at the contribution stage, I saw the Wagtail project #1 where I could contribute using my Technical documentation skills, and learn from amazing people.
+
+- [GitHub Profile](https://github.com/Otakuwind)
+- [Personal Website](https://0xwind.hashnode.dev)
+- [Twitter](https://twitter.com/0x_wind)
+- [LinkedIn](https://www.linkedin.com/in/edgar-nwajei-5822631b7)
```

Comment:
Not the biggest deal but the contributor guide does specifically ask for a _table_ with links